ci/scripts/hang_webserver.py

```diff
@@ -4,18 +4,35 @@
 from socketserver import ThreadingMixIn
 import threading
 import time
+import json
 
 class HTTPRequestHandler(BaseHTTPRequestHandler):
+    invocations = 0
+
     def do_POST(self):
-        if self.path.startswith('/403'):
+        if self.path.startswith('/reset'):
```

Comment:
we should consider if we want to keep our `hang_webserver` consistent with other drivers, because these seem to branch out rather hard lately

Reply:
I think for now this is the simplest option to mock errors/timeouts from SF, without any new tools.